.vortex/docs/questions-processed.md

```
@@ -0,0 +1,598 @@
+template/distribution files), and how these files relate to their non-dist
+counterparts.
+
+---
+
+## 29. Installation from Specific Commit
+
+**Original Question:**
+> How to install from a specific commit
+
+**Provided Answer:**
+> --uri=https://github.com/drevops/vortex.git@abcd123
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)